specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/examples/CheckQuotaAvailability.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "api-version": "2019-08-09-preview",
+    "location": "eastus"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "hostsRemaining": {
+          "gp": 0,
```

Comment:
>gp [](start = 11, length = 2)

It's not clear what `gp` or `he` mean. You should avoid acronyms in property names.

Reply:
This matches what is in production for this API version. It was reviewed and merged 12 days ago in https://github.com/Azure/azure-rest-api-specs/pull/9244 . I'll add it to the list of changes for the next API version.